packages/map-template/src/components/BottomSheet/BottomSheet.jsx

```diff
@@ -21,12 +21,12 @@ const BOTTOM_SHEETS = {
  * @param {Object} props.setCurrentLocation - The setter for the currently selected MapsIndoors Location.
  * @param {Object} props.currentCategories - The unique categories displayed based on the existing locations.
  * @param {function} props.onLocationsFiltered - The list of locations after filtering through the categories.
- * @param {function} props.onHideFloorSelector - Trigger the visibility of the floor selector to be hidden.
- * @param {function} props.onShowFloorSelector - Trigger the visibility of the floor selector to be shown.
  * @param {function} props.onDisableLocations - Restrict the user from interacting with the locations when in directions mode.
  * @param {function} props.onEnableLocations - Allow the user to interact with the locations when outside of directions mode.
+ * @param {function} props.onDirectionsOpened - Check if the directions page state is open.
+ * @param {function} props.onDirectionsClosed - Check if the directions page state is closed.
  */
-function BottomSheet({ currentLocation, setCurrentLocation, currentCategories, onLocationsFiltered, onHideFloorSelector, onShowFloorSelector, onDisableLocations, onEnableLocations}) {
+function BottomSheet({ currentLocation, setCurrentLocation, currentCategories, onLocationsFiltered, onDisableLocations, onEnableLocations, onDirectionsOpened, onDirectionsClosed}) {
```

Comment:
Do we need both the pairs for locations and directionsOpened? Can they be combined into one?

Reply:
For the disable/enable locations?

Reply:
I was wondering if we need all four:

`onDisableLocations, onEnableLocations, onDirectionsOpened, onDirectionsClosed}`

Are they (pairwise) handling the same thing? Not sure.

Reply:
That is true actually, the enable/disable location is something that is dependent on the state of directions, so I refactored the code and removed the unnecessary parameters.